docs/misc/misc.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Miscellaneous information
+has_children: true
+nav_order: 1
+permalink: /docs/miscellaneous-information
+---
+
+Miscellaneous useful information when working with this project
```

Comment:
Not sure I get the purpose of this file... is it just a placeholder for future info, or is this supposed to be a descriptor for the misc docs folder?

Reply:
I also am not sure about calling this something as general as `misc`/`miscellaneous-information`, as I think there could be a clearer name to use here. Maybe something more along the lines of `data` or `example data` or `testing`?

Reply:
> Not sure I get the purpose of this file... is it just a placeholder for future info, or is this supposed to be a descriptor for the misc docs folder?

This file will get displayed, but then act like a parent directory in the github pages file structure like **Getting started** here:
<img width="224" alt="Screen Shot 2020-01-02 at 10 00 49 AM" src="https://user-images.githubusercontent.com/10775718/71673441-d6078880-2d46-11ea-8a0e-1ac56870a721.png">

Reply:
@kabeaty I like the idea of changing this to `example data` 👍